x/claim/keeper/mission.go

```diff
@@ -0,0 +1,107 @@
+package keeper
```

Comment:
I see we scaffolded mission like a list. A mission is actually identified by an ID but there will be no notion of "appending" a mission and generating the ID on the fly since the list of mission is static and defined at genesis
I suggest removing methods related to this logic:
```
GetMissionCount
SetMissionCount
AppendMission
```